data/tribes/initialization/empire/units.lua

```diff
@@ -1268,20 +1473,58 @@ wl.Descriptions():new_tribe {
       {
          name = "empire_vineyard",
          helptexts = {
+            lore = latin_lore(
+               -- TRANSLATORS: DO NOT TRANSLATE, but you may transliterate into non-latin fonts.
+               --    Classical Latin quote meaning:
+               --    "Good wine needs no bush." (meaning no advertising)
+               _("Vino vendibili hedera non opus est."),
+               -- TRANSLATORS: Lore helptext for an empire production site: Vineyard
+               --    Translation for the classical Latin quote:
+               --    "Vino vendibili hedera non opus est."
+               --    Bush (ivy in the original) refers to the ivy leaves used for advertising
+               --    by places selling wine
+               _("Good wine needs no bush.")
```

Comment:
well, we can not assume anybody to know that a bush of ivy was a sign for a wine tavern. (in Germany for example  the equivalent is a broom. 
so I suggest the following
```suggestion
               _("Good wine needs no sign.")
```

Reply:
I fully agree, that's why I put all these explanations in the translators note, so that everybody can easily identify the traditional form in their own language. But unfortunately this seems to be the traditional phrase in English... (in hungarian the traditional form is already the equivalent of sign)

Reply:
ok good to go then.

many thanks